### PR TITLE
added front end queries

### PR DESCRIPTION
**Pull request by melinamboedecker**

### CODE REVIEW

File: client/src/utils/queries.js [hunk 1]
```diff
@@ -1 +1,161 @@
...
+export const QUERY_ALL_CATEGORIES = gql`
+    query categories {
+        categories {
+            _id
+            name
+    }
+}
+`;
+
+//For below query, do we need to put the posts in the Category typeDef?
```
Comment (vasylynash) [inline client/src/utils/queries.js:122]: I've added posts to category model. We'll see if it works

File: client/src/utils/queries.js [hunk 2]
```diff
@@ -1 +1,161 @@
...
+export const QUERY_ALL_LABELS = gql`
+    query labels {
+        labels {
+            _id
+            name
+    }
+}
+`;
+
+//For below query, do we need to put the posts in the Label typeDef?
```
Comment (vasylynash) [inline client/src/utils/queries.js:147]: I've added posts to the label model. We'll see if it works

**Review by vasylynash — APPROVED** [review]

File: client/src/utils/queries.js [hunk 3]
```diff
@@ -1 +1,161 @@
 import { gql } from "@apollo/client";
```
Comment (vasylynash) [inline client/src/utils/queries.js]: We use single quotes for JS. Update please

**Review by vasylynash — CHANGES_REQUESTED** [review]

**Review by Dimavich — APPROVED** [review]

**Review by Zoneam — APPROVED** [review]

File: client/src/utils/queries.js (hunk 3 above)
Comment (melinamboedecker) [reply client/src/utils/queries.js]: Okay, done

**Review by vasylynash — APPROVED** [review]

**Comment by melinamboedecker** [comment]
[front end react queries and mutations](https://trello.com/c/ebutCgrN/6-front-end-react-queries-and-mutations)

**Pull request merged**